styles/base/_links.scss

```diff
@@ -58,13 +57,32 @@
   &:hover,
   &:active,
   &:focus {
-    transition: $sprk-link-transition;
     color: $sprk-link-light-hover-color;
     font-weight: $sprk-link-light-hover-font-weight;
     border-bottom: $sprk-link-light-hover-border-bottom;
   }
 }
 
+/// Used when an inline link is on the dark background of the footer. There will be an underline on hover.
+.sprk-b-Link--footer {
+  color: $sprk-link-footer-color;
+  border-bottom: $sprk-link-footer-border-bottom;
+  font-weight: $sprk-link-footer-font-weight;
+
+  &:visited {
+    border-bottom: $sprk-link-footer-visited-border-bottom;
+  }
+
+  &:hover,
+  &:active,
+  &:focus {
+    color: $sprk-link-footer-hover-color;
+    font-weight: $sprk-link-footer-hover-font-weight;
+    border-bottom: $sprk-link-footer-hover-border-bottom;
+    cursor: $sprk-link-footer-hover-cursor;
```

Comment:
Added this cursor style because after testing on chrome and safari, hovering over a link within a block of text doesn't change the cursor. 
Before:
![IMG_5018](https://user-images.githubusercontent.com/4342363/118047502-d8b9bb00-b348-11eb-802b-f53cd0067383.JPG)

After:
![IMG_5019](https://user-images.githubusercontent.com/4342363/118047511-dc4d4200-b348-11eb-89cc-a22978428064.JPG)